src/main/java/be/uantwerpen/minelabs/entity/BohrBlueprintEntity.java

```diff
@@ -125,14 +127,15 @@ private void logicalTick() {
     }
 
     // Called by subatomic particle when it collides with this entity.
-    public void onParticleCollision(SubatomicParticleEntity particle) {
+    public void onParticleCollision(SubatomicParticleEntity particle, ServerPlayerEntity thrower) {
```

Comment:
I think you can just use the `owner` of the particle entity instead of having to pass the player along.

Reply:
Ah yes, didnt realise. Fixed.